tls/s2n_security_policies.c

```diff
@@ -439,8 +439,8 @@ const struct s2n_security_policy security_policy_null = {
     .minimum_protocol_version = S2N_TLS10,
```

Comment:
Sigh...We shouldn't be exposing `security_policy_null` to our customers at all, this exists because it replaced `cipher_preferences_null` which was introduced a long time ago for testing purposes.  I'll clean this up in a new PR.